it/utils_for_tests.go

```diff
@@ -216,6 +214,18 @@ func SetupApp(t *testing.T, bundle *smith_v1.Bundle, serviceCatalog, createBundl
 	require.NoError(t, err)
 
 	defer func() {
+		cli := smithClient.SmithV1().Bundles(cfg.Namespace)
+		b, err := cli.Get(bundle.Name, meta_v1.GetOptions{})
+		if err == nil {
+			if len(b.Finalizers) > 0 {
+				t.Logf("Removing finalizers from Bundle %q", bundle.Name)
+				b.Finalizers = nil
```

Comment:
It's annoying to remove finalizers manually to make the namespace go away if the test exits abruptly.